topics/ecology/tutorials/species-distribution-modeling/tutorial.md

```diff
@@ -0,0 +1,202 @@
+---
+layout: tutorial_hands_on
+topic_name: sdm
+tutorial_name: species distribution modeling
+---
+
+# Introduction
+{:.no_toc}
+
+Species Distribution Modeling (SDM) can help understand the distribution of a species depending on its environment. It can also attempt to quantify the impact of climate change on the species habitat, direct conservation efforts and predict invasive species distributions. This is done by associating data of species occurences (observations) with a set of environmental data (such as temperature and precipitation).
+
+The goal of this tutorial is to model a theorical ecological niche and predict species distribution in a future climate scneario by using SDM with the Wallace interactive environment on Galaxy. We'll use the data occurrences of US *Chrysemys picta* ([Painted turtle](https://en.wikipedia.org/wiki/Painted_turtle)) from the North America region.
+
+# Step 1: Loading a dataset
+
+In this study the datasets are all imported from the tool `Get species occurrences data and taxref informations` in Galaxy-E. With this tool, data are available from differents databanks like [GBIF](https://www.gbif.org/), [bison](https://www.gbif.org/), [iNaturalist](https://www.inaturalist.org/) and others.
+
+>    > ### {% icon tip %} Tip: Importing data set from a data bank
+
+>    > * Go into the "upload files" tool section (top left panel) then select the "Get species occurrences data and taxref informations" tool
+>    > * Fill the "Scientific name" with `Chrysemys picta`
+>    > * Choose the data source `gbif` and set the number of occurrences on `10000`
+>    > * Set "Get species information from taxref database" to "No"
+>    > * Click on "Execute"
+
+You should now have a file with about `9562` occurrences.
+
+Because you only need informations about occurrences and their location:
+> use the tool `Couper des colonnes d'un jeu de données tabulé`
+> then in "Couper les colonnes" type `c1,c2,c3,c44`
+
+>    > ### {% icon question %} Questions
+>    >
+>    > 1. For what stand c1, c2, c3, c44 ?
+>    >
+>    >    <details>
+>    >    <summary>Click to view answers</summary>
+>    >    <ol type="1">
+>    >    <li>c1 is the species name and c2 &c3 are respectively longitude an latitude corresponding to each occurrences of the file. The fourth column contain country code to have a possibility to easily filter occurences by countries.</li>
+>    >    </ol>
+>    >    </details>
+>    {: .question}
+
+Then, as we want to keep only occurence records from US, we will use a tool to filter the data based on the fourth column. Can you find a way to do that alone?
```

Comment:
Is it part of the question box? What about the answers below?

Reply:
no part of the question box